infrastructure-as-code/aws-lambda/README.md

```diff
@@ -0,0 +1,8 @@
+# Terraforming EC2 lifecycles with AWS Lambda
```

Comment:
I might make the filename `aws-lambda` a bit more specific as eventually we'll have many AWS Lambda guides. Maybe `ec2-lifecycles-aws-lambda` or something.

Reply:
Ok, i can do that.